README.md

```diff
@@ -265,6 +265,8 @@ elasticsearch::script { 'myscripts_dir':
 By default templates use the top-level `elasticsearch::api_*` settings to communicate with Elasticsearch.
 The following is an example of how to override these settings:
 
+Both legacy templates api and new composable templates api are supported using different data types.
```

Comment:
can you maybe extend the description a bit and mention the new types/defined resources? maybe also mention which elasticsearch versions are supported?

Reply:
I've expanded it a bit, let me know if there is still something missing/unclear

Reply:
thanks, looks good now.